packages/web-ui/src/styles/globals.css

```
@@ -1,8 +1,128 @@
-/*
- * Shared globals.css stub for @packrat/web-ui.
- *
- * A future PR will move the base layer + theme tokens currently duplicated
- * between apps/landing/app/globals.css and apps/guides/app/globals.css here,
- * so both Next.js apps can `@import '@packrat/web-ui/styles/globals.css'`
- * and only keep app-specific overrides locally.
- */
+@tailwind base;
```

Stylelint (17.6.0)

[error] Unexpected unknown at-rule "@tailwind" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)